mbuild/formats/gomc_conf_writer.py

```diff
@@ -4597,12 +4597,9 @@ def write_conf_file(self, conf_filename):
                 "{:25s} {}\n".format("Rswitch", self.Rswitch)
             )
         data_control_file.write("{:25s} {}\n".format("Exclude", self.Exclude))
-        if self.VDWGeometricSigma is True:
-            data_control_file.write(
-                "{:25s} {}\n".format(
-                    "VDWGeometricSigma", self.VDWGeometricSigma
-                )
-            )
+        data_control_file.write(
```

Comment:
I just wanted to have it print either way, aside from the fact the VDWGeometricSigma = False by default.  It may be better for new GOMC users to just have it printed.  Additionally, it was in the test cases, and it is easier to check if it is printed, instead of checking if it is not there.